### ISSUE TITLE
[BUG][WCAG] requests/new - Little to no feedback on buttons to indicate focus. 

### ISSUE THREAD

**Issue by j-isenberg** [reach4help/reach4help#647]
**Describe the bug**
When the user hovers over or tabs into certain buttons on the requests/new page, there is either not enough of a change to the button or no change at all to indicate focus. 
All elements with user interaction must indicate when it is in focus, whether by hovering over it with the cursor, or tabbing into the element with the keyboard. 

**1. Use my location:** The colour does not change enough to be considered compliant when the user hovers over or tabs into this button. 
Idle hex code: 7D00A3
Focused hex code: A12596
Minimum contrast requirement: 3:1

**2. "Menu", "New request" and "Notification" buttons:** There is NO feedback on any of these buttons. 

**Optional. "Search" and "Cancel" button text:** The colour of the text does not have a very high contrast when hovered over. This is optional because the boarder of these buttons do change enough. 

https://reach4help-staging-web-client.firebaseapp.com/requests/new

**To Reproduce**
1. Navigate to requests/new page. 
2. Hover over each button.
3 (w/ Axe). Check hex codes of idle and focused buttons. 
4. Notice that contrasts do not meet minimum requirements. 
5. Notice that focusing over the menu, new request and notification buttons does nothing. 

**Expected behavior**
Users are given proper feedback to indicate focus. Feedback include compliant contrast ratios. 

**Screenshots**
Video posted in Slack team-user-testing channel. 
https://slack-files.com/T010H81FF60-F015MQLBQ48-620cd65262

**Desktop (please complete the following information):**
- Windows 10 - Chrome 

**Additional context**
Two-color focus indicator techniques: https://www.w3.org/WAI/WCAG21/Techniques/css/C40
https://contrastchecker.com/ 
https://www.w3.org/WAI/WCAG21/quickref/#focus-visible

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/647)

*This information is updated automatically. To modify it, please use ZenHub.*

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/647)

*This information is updated automatically. To modify it, please use ZenHub.*

**Comment by j-isenberg** [comment]
Most buttons on the Request/new page now gives the user feedback when hovered over or tabbed into. 
Please note that while the main Menu buttons do have focus indicators, the X to close the drawer does not. 
Also note that the Menu and Notification drawers do not have keyboard functionality. This ticket cannot be fully checked or closed until https://github.com/reach4help/reach4help/issues/649 is resolved.
